src/PublishCodeCovCoverageTask/__tests__/index.test.ts

```
@@ -758,7 +769,24 @@ describe('PublishCodeCovCoverage', () => {
+    // Check that verifyFileChecksum was called with the correct arguments
+    expect(verifyFileChecksum).toHaveBeenCalledWith(
+      'codecov',
+      'codecov.SHA256SUM',
+      expect.any(Function)
+    );
+    // Advanced check: the logger function should delegate to logger.info
+    const call = (verifyFileChecksum as jest.Mock).mock.calls.find(
+      ([file, sum, loggerFn]) => file === 'codecov' && sum === 'codecov.SHA256SUM'
+    );
+    expect(call).toBeTruthy();
+    const loggerFn = call && call[2];
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)